specs/MultiProfile.md

```diff
@@ -509,13 +662,26 @@ namespace Microsoft.Web.WebView2.Core
 
         String ProfilePath { get; };
 
+        
         CoreWebView2CookieManager CookieManager { get; };
         
         [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Profile7")]
         {
             void Delete();
             event Windows.Foundation.TypedEventHandler<CoreWebView2Profile, Object> Deleted;
         }
+        
+        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Profile2")]
```

Comment:
We already got up to Microsoft.Web.WebView2.Core.ICoreWebView2Profile7 above.  Shouldn't new interfaces be 8, 9, ...?

Reply:
Yes, will update the number to the latest correct state when promoting these to public.